woocommerce/woocommerce-gutenberg-products-block
# Implement "return to cart" and "place order" buttons on checkout

Adds a return to cart link, and the place order button.

**Return to cart**
- Can be disabled in block options
- URL can be controlled in block options
- URL defaults to cart page set in WC

**Place order**
- Standard button with styling and callback
- Gets the callback action and text from checkout context
- I have added a TODO for the actual action. The button is non-functional in this PR

**Misc**
Whilst styling these new buttons, I also styled some of the margins on the rest of the form to match the mockup. This was mostly needing to align things vertically.

Fixes #1303

### Screenshots

![Screenshot 2020-03-10 at 14 19 34](https://user-images.githubusercontent.com/90977/76321623-316f6d00-62da-11ea-985a-4a98ae623a64.png)
![Screenshot 2020-03-10 at 14 19 17](https://user-images.githubusercontent.com/90977/76321624-32080380-62da-11ea-93b9-4ee8d2431733.png)

### How to test the changes in this Pull Request:

1. Go to the checkout
2. Check appearance of link and button
3. Edit block (might be invalidated)
4. Confirm you can toggle the cart link on/off and change the URL.
